NDTensors/ext/NDTensorsCUDAExt/set_types.jl

```diff
@@ -1,18 +1,14 @@
 # TypeParameterAccessors definitions
 using CUDA: CUDA, CuArray
-using NDTensors.TypeParameterAccessors: TypeParameterAccessors, Position
+using NDTensors.TypeParameterAccessors:
+  TypeParameterAccessors, Position, default_type_parameters
 using NDTensors.GPUArraysCoreExtensions: storagemode
+using GPUArraysCore: AbstractGPUArray
 
-function TypeParameterAccessors.position(::Type{<:CuArray}, ::typeof(eltype))
-  return Position(1)
-end
-function TypeParameterAccessors.position(::Type{<:CuArray}, ::typeof(ndims))
-  return Position(2)
-end
 function TypeParameterAccessors.position(::Type{<:CuArray}, ::typeof(storagemode))
   return Position(3)
 end
 
 function TypeParameterAccessors.default_type_parameters(::Type{<:CuArray})
-  return (Float64, 1, CUDA.Mem.DeviceBuffer)
+  return (default_type_parameters(AbstractGPUArray)..., CUDA.Mem.DeviceBuffer)
```

Comment:
```suggestion
  return (default_type_parameters(AbstractArray)..., CUDA.Mem.DeviceBuffer)
```

Reply:
(And if so, don't load `GPUArraysCore`.)

Reply:
Same as above, thanks!